src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs

```diff
@@ -75,6 +72,12 @@ internal class DebugService
         /// </summary>
         public DebuggerStoppedEventArgs CurrentDebuggerStoppedEventArgs { get; private set; }
 
+        /// <summary>
+        /// Returns a task that completes when script frames and variables have completed population
+        /// </summary>
+        public Task StackFramesAndVariablesFetched { get; private set; }
+
+
```

Comment:
```suggestion
```

Extra line, hopefully this suggestion works

Reply:
Didn't work in vscode but I fixed it anyways.